site/content/docs/user/exposing-applications.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Accesing applications inside the cluster"
+menu:
+  main:
+    parent: "user"
+    identifier: "exposing-applications"
+    weight: 1
+---
+# Accesing services and applications inside a kubernetes cluster
+
+If you need a kubernetes cluster to develop or to test your applications, services, charts, ... Minikube is a great tool but lacks the capability to create multi-node clusters.
+
+`kind` use docker to create to create multi-nodes cluster, despite the great benefits of docker it has the problem that the implementation differ for each OS, and [Windows][[Docker for Windows networking]] and [Macs][Docker for Mac networking] have important networking limitations. Main issue is that you can't reach the cluster node internal ip addresses.
+
+This guide covers how to access your pods and services inside your kubernetes cluster once you have deployed them with `kind`. We really suggest you to get familiar with the [kubernetes concepts for exposing applications][kubernetes exposing services] and [when to use what][kubernetes external access].
+
+For Linux users you can follow this nice write up done by @mauilion using [MetalLB Load Balancer][kind metallb].
+
+The rest of the document will be based on Mac OSX and Windows with WSL (Windows Subsystem for Linux)
+
+## Create the Kubernetes cluster
+
+Use a configuration file to create the kuberntes cluster 
+`config-ipv4-ci.yaml`
+
+```yaml
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+nodes:
+# the control plane node
+- role: control-plane
+- role: worker
+- role: worker
+````
+
+And create the cluster
+
+```sh 
+$ kind create cluster --config config-ipv4-ci.yaml
+Creating cluster "kind" ...
+ ✓ Ensuring node image (kindest/node:v1.14.1) 🖼
+ ✓ Preparing nodes 📦📦📦
+ ✓ Creating kubeadm config 📜
+ ✓ Starting control-plane 🕹️
+ ✓ Installing CNI 🔌
+ ✓ Installing StorageClass 💾
+ ✓ Joining worker nodes 🚜
+Cluster creation complete. You can now use the cluster with:
+
+export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
```

Comment:
`--name="kind"` is not needed as it's the default. If you want to be more explicit then it should be added to `kind create cluster --config config-ipv4-ci.yaml` as well to be consistent :)

Reply:
this is a copy paste from the output of the `kind` command 😅